exporter/exporter_test.go

```diff
@@ -223,6 +224,11 @@ func TestImportingUsersGroupsSecretScopes(t *testing.T) {
 			meAdminFixture,
 			repoListFixture,
 			emptyGitCredentials,
+			{
+				Method:   "GET",
+				Resource: "/api/2.0/workspace/list?path=%2F",
+				Response: workspace.ObjectList{},
+			},
```

Comment:
move into a separate fixture, like it's done for `emptyGitCredentials`, etc.?

Reply:
a bit of overkill for 2 repetitions, but fine